src/plugins/home/public/application/components/guided_onboarding/use_case_card.scss

```diff
@@ -0,0 +1,27 @@
+.gettingStarted__useCaseCard {
+  // on smaller screens make top image taller
+  @include euiBreakpoint('xs','s') {
+    min-height: 250px;
+  }
+  // on medium screens make it lower
+  // euiBreakpoint m
+  min-height: 150px;
+  // on larger screens taller but not too tall
+  @include euiBreakpoint('l','xl') {
+    min-height: 200px;
```

Comment:
The beauty of Emotion is the ability to do these calculations in JS...

Reply:
I'm struggling with `@include euiBottomShadow;` on line 26, is that possible to do with Emotion? I was also not able to find any Kibana docs about guidelines for Emotion usage, can you please recommend something?

Reply:
We do have all our shadows in emotion (they're not named 1:1 but use a better t-shirt sizing): https://elastic.github.io/eui/#/theming/more-tokens#shadow?themeLanguage=js

`euiBottomShadow` would be medium `useEuiShadow('m')` or `euiShadow(euiThemeContext, 'm')`

Reply:
@cchaos I don't think the import from `global_styling/mixins` works either. I'm getting this error `Module not found: Error: Can't resolve '@elastic/eui/src/themes/amsterdam/global_styling/mixins'`

Reply:
Are you up to date with `main`'s version of EUI?

Reply:
Very odd, can you try to auto-import?